.travis.yml

```diff
@@ -7,8 +7,9 @@ go:
   # - https://github.com/alecthomas/gometalinter/issues/358
   # - https://github.com/golang/go/issues/21712
   # TODO jiayu: re-evaluate the pending issues and add back `gotype`
-  - 1.8
-  - 1.9
+  - "1.8"
+  - "1.9"
+  - "1.10"
 
```

Comment:
@jlewi https://github.com/travis-ci/travis-ci/issues/9247

fixed for all above